src/app/components/Application/Application.jsx

```diff
@@ -71,6 +38,36 @@ class App extends React.Component {
 
   componentDidMount() {
     Store.listen(this.onChange);
+    // Listen to the browser's navigation buttons.
+    this.props.route.history.listen((location) => {
```

Comment:
yay!!!!

Reply:
I am sorry but I got this error when load the page

`Uncaught TypeError: Cannot read property 'action' of undefined`

Reply:
This seems to be fixed by giving the listener a default location object to start with